src/main/java/uk/gov/hmcts/reform/sscs/utility/JsonDataReader.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.sscs.utility;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.io.InputStream;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ClassPathResource;
+
+@Slf4j
+public class JsonDataReader {
+
+    private JsonDataReader() {
+
+    }
+
+    public static InputStream getResource(String filename) throws IOException {
+        return new ClassPathResource(filename).getInputStream();
+    }
+
+    public static <T> T importObjectDataList(String filename, TypeReference<T> typeReference) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper();

```

Comment:
Why ObjectMapper isn't a static final instead of creating one each time?  Or better, inject it.
Why does this class need to be a singleton?

Reply:
What do you mean by injecting it?
This class is a helper